inc/assets.php

```diff
@@ -130,3 +112,26 @@ function enqueue_frontend_scripts() : void {
 
 	wp_enqueue_script( 'byg-frontend' );
 }
+
+/**
+ * Enqueue block editor assets.
+ *
+ * @return void
+ */
+function enqueue_block_editor_assets() : void {
+	register_build_asset(
+		'byg-editor',
+		'before-you-go-editor.js',
+		/**
+		 * Allow a theme or plugin to integrate with this plugin by registering
+		 * a custom script as a dependency of our BYG frontend script. If the
+		 * dependency script defines a BYG global with a custom callback or
+		 * trigger, that will alter the behavior of the BYG activation.
+		 *
+		 * @param string[] $dependencies Script handles to load ahead of the frontend bundle.
+		 */
+		apply_filters( 'byg\editor_script_dependencies', [ 'altis-analytics-audience-ui' ] )
```

Comment:
I'm going to remove this filter as it feels overcomplicated, now that we're relying on Altis Analytics for the bulk of the logic. It's not too cumbersome to alter a registered script and add a new dependency after the fact, so having this filter in place feels unnecessary.